pkg/test/case.go

```diff
@@ -196,6 +196,7 @@ func shortString(obj *corev1.ObjectReference) string {
 func (t *Case) Run(test *testing.T, tc *report.Testcase) {
 	ns := t.determineNamespace()
 	if err := t.CreateNamespace(ns); err != nil {
+		tc.Failure = report.NewFailure(err.Error(), nil)
```

Comment:
Should we use AddFailure here?

Reply:
the `AddFailure` (renamed to `SetFailure`) is at the TestSuites (collection level).  This failure is at the testcase level and will report a failure for each test case if all namespaces can not be created).  It might be debatable if we need to report each one individual.. but if we want to fail the whole suite on 1 namespace creation issue we would need to report from this run... and we currently don't